src/PIL/ImageFont.py

```diff
@@ -674,6 +683,10 @@ def getmask2(
                  :py:mod:`PIL.Image.core` interface module, and the text offset, the
                  gap between the starting coordinate and the first marking
         """
+        if fill is not _UNSPECIFIED:
+            deprecate("fill", 10)
+        else:
+            fill = Image.core.fill
```

Comment:
Can we flip so the conditional is positive?

```suggestion
        if fill is _UNSPECIFIED:
            fill = Image.core.fill
        else:
            deprecate("fill", 10)
```